docs/testrunner-toolkit/saucectl.md

```diff
@@ -415,3 +415,72 @@ Enables detailed output during the test run in order to facilitate troubleshooti
 ```bash
 saucectl run --verbose
 ```
+
+## `$ saucectl completion`
+
+Allows you to generate completion script for `bash`, `zsh`, `fish` and `powershell` shells.
+
+### bash
+
+#### Linux
+
+```bash
+saucectl completion bash > /etc/bash_completion.d/saucectl
+```
+---
+
+#### macOS
+
+```bash
+saucectl completion bash > /usr/local/etc/bash_completion.d/saucectl
+```
+---
+
+### zsh
+
+If shell completion is not already enabled in your environment, you will need to enable it.
+You can execute the following once:
+
+```bash
+$ echo "autoload -U compinit; compinit" >> ~/.zshrc
+```
+---
+  
+To load completions for each session, execute once:
+
+```bash
+saucectl completion zsh > "${fpath[1]}/_saucectl"
+```
+---
+
+You will need to start a new shell for this setup to take effect.
+
+### fish
+
+```bash
+saucectl completion fish | source
+```
+---
+
+To load completions for each session, execute once:
+
+```bash
+saucectl completion fish > ~/.config/fish/completions/saucectl.fish
+```
+---
+
+### PowerShell
+
+```bash
+saucectl completion powershell | Out-String | Invoke-Expression
+```
+---
+
+To load completions for every new session, run:
+
+```bash
+saucectl completion powershell > saucectl.ps1
+```
+---
+
+and source this file from your PowerShell profile.
```

Comment:
```suggestion
```
I am afraid this last statement might get lost under the example. Have I written it to make sense above?

Reply:
I fixed this issue.